.env_SAMPLE

```diff
@@ -33,15 +33,16 @@ export ALLOW_ADMIN_URL=True
 #export AKAMAI_HOST=<akamai_hostname>
 
 ############################
-# Mysql Database Server.
+# MySQL Database Server
 ############################
 
 export MYSQL_NAME=v1
+export MYSQL_HOST=localhost
```

Comment:
@contolini and I discovered that `create-mysql-db.sh` required this env var to be set.